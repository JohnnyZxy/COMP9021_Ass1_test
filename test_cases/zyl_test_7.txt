Yesterday, I was playing League of Legends. 
Sir Darius was solo, and Sirs Garen, Jinx and Lee were duo.
Sir Darius said: "I am a Knight."
But Sir Riven disagreed, "Darius and Lee are Knave!"
Sir Jinx whispered: "Garen or Riven is a Knight."
What do you think? 