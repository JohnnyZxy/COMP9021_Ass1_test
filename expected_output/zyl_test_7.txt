The Sirs are: Darius Garen Jinx Lee Riven
There are 8 solutions.